src/.github/workflows/pre-commit.yml.jinja

```diff
@@ -15,19 +15,22 @@ jobs:
     runs-on: ubuntu-20.04
 {%- else %}
     runs-on: ubuntu-22.04
+{%- endif %}
+{%- if odoo_version < 11 %}
+    container:
+      image: python:2.7.18-buster
 {%- endif %}
     steps:
       - uses: actions/checkout@v3
+{%- if odoo_version >= 11 %}
       - uses: actions/setup-python@v2
-{%- if odoo_version < 11 %}
-        with:
-          python-version: "2.7"
-{%- elif odoo_version < 13 %}
+{%- if odoo_version < 13 %}
         with:
           python-version: "3.6"
 {%- elif odoo_version < 14 %}
         with:
           python-version: "3.8"
+{%- endif %}
```

Comment:
We should pin the python version for all Odoo versions. But that's for another PR, I guess.

Reply:
It is, but the PR was closed.

https://github.com/OCA/oca-addons-repo-template/pull/196